### ISSUE TITLE
Optional multiple choice items should be deselectable completly

### ISSUE THREAD

**Issue by mpw-wwu** [rubenarslan/formr.org#377]
If a multiple choice item is set to optional, it should be possible to unset the radio button group after it has been clicked once.

**Comment by rubenarslan** [comment]
But that's not standard for HTML forms. I've also not yet seen a user story supporting this. I think making it easy to set up a visually distinct "I don't know/don't want to answer" option is more urgent.

**Comment by mpw-wwu** [comment]
It is non standard, that's true. But it leads to different outcome depending on whether someone clicked the item in the first place or didn't.

**Comment by rubenarslan** [comment]
I agree. So, I think UI/UX-wise it makes sense if you use an optional radio button type item, you should have a "non-response" category to select if you don't make a choice. When implementing this, we need to think about the interface for the non-response choices. I first thought: when the item is optional, let's turn the last choice category into a separate button for non-response. But sometimes we need multiple choices. Another option could be recognising non-response options by using special values, but what would those be? Negative values are a common standard, but I'd rather not go that route, because they can also be real. Mixing text labels with numeric responses could be an extensible option, i.e.

| list_name | value | label | 
| -------- | ------ | ----- |
| likert3 | 1 | low | 
| likert3 | 2| medium | 
| likert3 | 3 | high | 
| likert3 | doesnt_apply | doesn't apply to me | 
| likert3 | dont_know | don't know |

**Comment by rubenarslan** [comment]
Kept open for form_v2: the v1 renderer is no longer being refined, and the form_v2 branch's button-group component carries deselect handling for optional single-choice items.
